packages/algorand-js/src/interpreter/txn.ts

```diff
@@ -0,0 +1,118 @@
+import { AssetParamsEnc } from "algosdk";
+
+import { TealError } from "../errors/errors";
+import { ERRORS } from "../errors/errors-list";
+import { TxFieldDefaults } from "../lib/constants";
+import { toBytes } from "../lib/parse-data";
+import { StackElem, TxField, Txn, TxnFields, TxnType } from "../types";
+import { Interpreter } from "./interpreter";
+import { Op } from "./opcode";
+
+const assetTxnFields = new Set([
+  'ConfigAssetTotal',
+  'ConfigAssetDecimals',
+  'ConfigAssetDefaultFrozen',
+  'ConfigAssetUnitName',
+  'ConfigAssetName',
+  'ConfigAssetURL',
+  'ConfigAssetMetadataHash',
+  'ConfigAssetManager',
+  'ConfigAssetReserve',
+  'ConfigAssetFreeze',
+  'ConfigAssetClawback'
+]);
+
+// return default value of txField if undefined,
+// otherwise return parsed data to interpreter
+export function parseToStackElem (a: unknown, field: TxField): any {
```

Comment:
Please change return type to `StackElem`

Reply:
yes, sorry.